### DISCUSSION TITLE
[Support]: (0.14.0 beta1) Birdseye is frozen on a still frame from 3 hours ago.

### DISCUSSION THREAD

**Discussion by ksheyman** [blakeblackshear/frigate#11675] [General Support]
### Describe the problem you are having

I just pulled up my birdseye and found that it was frozen on a still image from 3 hours ago. The rest of my frigate server is working fine, including the camera the image is from.  Checking frigate logs I see this:

```
2024-05-31 19:33:08.790218545  [2024-05-31 19:33:08] frigate.output.birdseye        WARNING : Unable to copy frame kitchen1717187809.095321 to birdseye.

2024-05-31 19:33:08.906401799  [2024-05-31 19:33:08] frigate.output.birdseye        WARNING : Unable to copy frame kitchen1717187809.095321 to birdseye.

2024-05-31 19:33:09.018558898  [2024-05-31 19:33:09] frigate.output.birdseye        WARNING : Unable to copy frame kitchen1717187809.095321 to birdseye.
```

repeated thousands and thousands of times. 

I restarted the container and birdseye is now working properly. Not sure what happened there, but I figured you guys might want to know since it was the beta. I am using go2rtc if that matters, running in docker on Ubuntu. I'd be happy to do further investigating if you tell me what you need (and how to get it).

### Version

0.14.0 Beta 1

### Frigate config file

```yaml
working config before upgrade
```

### Relevant log output

```shell
see above
```

### FFprobe output from your camera

```shell
I can't get that to work, but the camera stream shows in frigate just fine.
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Amcrest

**Comment by SailingSailing** [comment]
Same issue for me. If I open the camera, it displays fine. But on the Birdseye view, it's frozen.
